Translation corrections (#91)
07th-mod/higurashi-console-arcs: Update/some_007.txt

//r行き交う人々の流れはみんな、手に持った傘で顔を隠して歩く。ぶつかっても、水滴が跳ねても相手を見ようとしない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "行き交う人々の流れはみんな、手に持った傘で顔を隠して歩く。ぶつかっても、水滴が跳ねても相手を見ようとしない。",

> 			NULL, "Crowds move to and fro, everyone walking while hiding their face with an umbrella in their hand. Even when they bumped into others or water bounced off them, they did didn't even try to look at one another.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Crowds move to and fro, everyone walking while hiding their face with an umbrella in their hand. Even when they bumped into others or water bounced off them, they didn't even try to look at one another.", Line_Normal);
```

	ClearMessage();

//rそれぞれの人が見えているのは、…ただ、自分の進む方向に開かれた、視野の狭い空間だけ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それぞれの人が見えているのは、…ただ、自分の進む方向に開かれた、視野の狭い空間だけ。",

> 			NULL, "All each of them can see is the narrow space forward in their line of sight.", Line_Normal);

Reviewer: ```suggestion
			NULL, "All any of them can see is the narrow space directly ahead in their line of sight.", Line_Normal);
```

//rまた、ため息。…考えてみれば最近、何かにつけてため息ばかりをついていることに気づく。それも次第に間隔が小さくなって、そして吐く息も重くなっている。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "また、ため息。…考えてみれば最近、何かにつけてため息ばかりをついていることに気づく。それも次第に間隔が小さくなって、そして吐く息も重くなっている。",

> 			NULL, "And I let out a sigh. ...I noticed that I've been sighing a lot recently. The intervals between them was getting shorter, and I've been exhaling heavier as well.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And I let out a sigh. ...I noticed that I've been sighing a lot recently. The intervals between them were getting shorter, and I've been exhaling heavier as well.", Line_Normal);
```

//rいつも温厚で優しいおばーちゃんだけど、…頑固すぎるとも思えるくらいに融通が利かないところがあるから、この対立状態はまだ、しばらく続くだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いつも温厚で優しいおばーちゃんだけど、…頑固すぎるとも思えるくらいに融通が利かないところがあるから、この対立状態はまだ、しばらく続くだろう。",
			NULL, "Grandma has always been kind and gentle, but... she can be too set in her ways for her own good, so this confrontation will likely continue for a while.", Line_Normal);
	ClearMessage();

//rそしてこういう時、…おとーさんは二人をとりなすどころか、情けないことに近づきもしない。だから代わりに、私が二人の間に立たされて連絡役というか、…まるで緩衝材みたいに扱われていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そしてこういう時、…おとーさんは二人をとりなすどころか、情けないことに近づきもしない。だから代わりに、私が二人の間に立たされて連絡役というか、…まるで緩衝材みたいに扱われていた。",

> 			NULL, "At times like this... Dad never mediates. He stays far away from the two of them and avoids them, pathetically enough. So instead, I'm the one treated like their mediator... or rather, their buffer", Line_Normal);

Reviewer: ```suggestion
			NULL, "At times like these... Dad never intervenes. He stays far away from the two of them and avoids them, pathetically enough. So instead, I'm the one treated like their mediator... or rather, their buffer.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100601", 256, TRUE);
	OutputLine(NULL, "「なんで、私ばっかり…」",
			NULL, "\"Why is it always me...\"", Line_Normal);
	ClearMessage();

//r苦い思いをかみ締めて、私は愚痴っぽく呟く。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "苦い思いをかみ締めて、私は愚痴っぽく呟く。",

> 			NULL, "I complain to myself as I reflect on those bitter thoughts.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I complained to myself as I reflected on those bitter thoughts.", Line_Normal);
```

//r泣きたいほどに重く濁った澱が、胸の中に溜まり続けて気持ちを沈ませる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "泣きたいほどに重く濁った澱が、胸の中に溜まり続けて気持ちを沈ませる。",

> 			NULL, "Heavy sediments filled my heart and drove me to depression and the brink of tears.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Heavy clouds filled my heart, driving me to depression and the brink of tears.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "入居者のみんなが就寝してからもそうだったし、休憩も交替制だ。だから、控え室に誰もいなくなるのはいっせいに人手が必要となる１日３回の食事時と、おやつの配膳の時だけだった。",
				NULL, "<size=-2>That's still the case even while the tenants are asleep, and they take breaks in shifts. So the only time that the waiting room is left empty is during the three meals of the day and snack time.", Line_Normal);
	} else {
		OutputLine(NULL, "入居者のみんなが就寝してからもそうだったし、休憩も交替制だ。だから、控え室に誰もいなくなるのはいっせいに人手が必要となる１日３回の食事時と、おやつの配膳の時だけだった。",

> 				NULL, "That's still the case even while the tenants are asleep, they take breaks in shifts. So, the only time that the waiting room is left empty is during the three meals of the day, where all available manpower is required to serve food.", Line_Normal);

Reviewer: ```suggestion
				NULL, "That's still the case even while the tenants are asleep, and they take breaks in shifts. So the only time that the waiting room is left empty is during the three meals of the day and snack time.", Line_Normal);
```

//r部屋にあるいろんな家具にも、余すところなくびっしりと貼られてある。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "部屋にあるいろんな家具にも、余すところなくびっしりと貼られてある。",
			NULL, "They were also firmly attached to every piece of furniture in the room.", Line_Normal);
	ClearMessage();

//rいったい、どうやってこれだけの数を揃えたんだろうかと恐ろしくなるほどに、もとは白かったはずの室内がすっかり紅に染まっていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いったい、どうやってこれだけの数を揃えたんだろうかと恐ろしくなるほどに、もとは白かったはずの室内がすっかり紅に染まっていた。",

> 			NULL, "Their number was so terrifyingly overwhelming that the formerly white room was completely dyed red.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Their numbers were so terrifyingly overwhelming that the formerly white room was completely dyed red.", Line_Normal);
```
